docs/structured-streaming-kafka-integration.md

```diff
@@ -373,11 +375,204 @@ The following configurations are optional:
 </tr>
 </table>
 
+## Producing Data to Kafka
```

Comment:
I think section needs a big caveat regarding the lack of guarantees of Kafka writes - since Kafka does not support transactions, data may be duplicated due to failures and reexecutions